interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
Sure, I think the splitting the Dockerfiles into two and making a `Dockerfile.base` makes sense. I guess we can totally rm -r the node_modules folder from our machines and drop it from the .dockerignore file here, as long as that does not interfere with people's development environments while we switch over to always having the Docker container build these files (even on our development machines). Do you think this makes sense @mhuyck?